src/lib/fetchChatter.ts

```diff
@@ -30,9 +30,9 @@ export class FetchChatter {
             'chatter');
     }
 
-    list(): Promise<any> {
```

Comment:
@kidtsunami it may be reasonable to pick out 10-12 "most used" chatter calls and implement them here ... just not sure how we'd be able to determine that.